build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/views/endpoints_module.rb

```diff
@@ -134,6 +134,8 @@ def built_in_client_context_param_value(param_data)
           else
             'context.config.use_dualstack_endpoint'
           end
+        when 'AWS::Auth::CredentialScope'
```

Comment:
I think we may also need updates to the endpoint_spec_class (eg: https://github.com/aws/aws-sdk-ruby/blob/version-3/build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/views/spec/endpoint_provider_spec_class.rb#L149)

Reply:
Yeah. I was going to wait for the actual rules but I can add it now.

Reply:
This change was premature